### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by cfox1455** [jrich523/NimblePowerShell#15]
Would love a way to pull the replication partners and their description field over as well.  Great job on everything!

**Comment by jrich523** [comment]
That is one of the things on the list, but, until im told they have a public REST API I wont be changing this. Go bug them about REST!

**Comment by cfox1455** [comment]
Yep, they have one in 2.3 but we can’t quite upgrade yet (large waiting cycle on our end for safety purposes)  thanks!


Christopher Fox
Chief Technology Officer
[cid:IdealIntegrations_ee791edc-fea4-4fa0-a108-e8f88152f341.png]
Ideal Integrations, Inc. | 800 Regis Avenue | Pittsburgh, PA 15236
E-Mail: cfox@idealintegrations.net<mailto:cfox@idealintegrations.net> | Phone: 412-349-6680 x1111 | Direct: 412-349-6666
Web: http://www.idealintegrations.net<http://www.idealintegrations.net/>  | Service Desk: 412-349-6678 | Fax: 412-349-6670
